docs/src/main/sphinx/release/release-427.md

```diff
@@ -0,0 +1,73 @@
+# Release 427 (13 Sep 2023)
+
+## Security
+
+* Added access control check to redirected tables when accessing table comments. ({issue}`18514`)
+
+## ClickHouse connector
+
+* Add support for pushing down `UPDATE` statements to ClickHouse. ({issue}`16445`)
+
+## Delta Lake connector
+
+* Add support for reading tables with deletion vectors. ({issue}`16903`)
+
+## Druid connector
+
+* Add support for pushing down `UPDATE` statements to Druid. ({issue}`16445`)
+
+## Hive connector
+
+* Add support for pushing down `UPDATE` statements to Hive. ({issue}`16445`)
+* Add support for `double` to `varchar` coercion in Hive tables. ({issue}`18930`)
+* Remove legacy Hive readers and writers. The `*_native_reader_enabled` and
+  `*_native_writer_enabled` session properties and `*.native-reader.enabled` and
+  `*.native-writer.enabled` configuration properties have been removed. ({issue}`18241`)
+* Remove support for S3 Select. The `s3_select_pushdown_enabled` session
+  property and the `hive.s3select*` configuration properties have been removed. ({issue}`18241`)
+* Remove support for disabling optimized symlink listing. The
+  `optimize_symlink_listing` session property and
+  `hive.optimize-symlink-listing` configuration property have been removed. ({issue}`18241`)
+* Fix incompatibility with Hive OpenCSV deserialization. As a result, when the
+  escape character is explicitly set to `"`, a backslash must be used instead. ({issue}`18918`)
+* Fix slow reading of CSV files on AWS S3. ({issue}`18976`)
+
+## Iceberg connector
+
+* Add support for usage of `date` and `timestamp` arguments in `FOR TIMESTAMP AS
+  OF` expressions. ({issue}`14214`)
+* Reduce coordinator memory usage when running queries on tables with large
+  metadata. ({issue}`19009`)
+
+## Kudu connector
+
+* Add support for pushing down `UPDATE` statements to Kudu. ({issue}`16445`)
+
+## MySQL connector
+
+* Add support for pushing down `UPDATE` statements to MySQL. ({issue}`16445`)
+
+## Oracle connector
+
+* Add support for pushing down `UPDATE` statements to Oracle. ({issue}`16445`)
+* Fix potential query failure when joins are pushed down to Oracle. ({issue}`18924`)
+
+## Phoenix connector
+
+* Add support for pushing down `UPDATE` statements to Phoenix. ({issue}`16445`)
+
+## PostgreSQL connector
+
+* Add support for pushing down `UPDATE` statements to PostgreSQL. ({issue}`16445`)
+
+## Redshift connector
+
+* Add support for pushing down `UPDATE` statements to Redshift. ({issue}`16445`)
+
```

Comment:
@vlad-lyutenko Ignte and SingleStore connector don't support UPDATE statement in #16445?

Reply:
Supported connectors:

Ignite
MariaDB
MySql
Oracle
Postgres
Redshift
Singlestore
SqlServer